jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -226,7 +235,21 @@ def apply_gpu_config(self, gpu_mode, gpu_count, pod):
         if gpu_mode == self.GPU_MODE_PRIVILEGED:
           pod.spec.security_context.privileged = True
 
-    return pod
+      if gpus:
+        for gpu in gpus:
+          tolerations.append(gpu.get('node_tolerations')[0])
```

Comment:
Assuming the idea of `gpus` being list is that there is `type` which in the future could be used to map back to GPU type, how would this work? You are attaching all the tolerations here..shouldn't just 1 `gpu` config item be used?